package.json

```diff
@@ -66,7 +66,8 @@
     "reselect": "^4.0.0",
     "rn-fetch-blob": "^0.10.15",
     "sanitize-html-react": "^1.13.0",
-    "styled-components": "^4.3.1"
+    "styled-components": "^4.3.1",
+    "url-parse": "^1.4.7"
```

Comment:
This needs further discussion. Imo we kind of need this because writing a custom url parser (or doing it using regex) is error prone.

This polyfill is also not perfect but acceptable.